frontend/app/views/spree/shared/_products.html.erb

```diff
@@ -19,7 +19,7 @@
 <% if products.any? %>
   <div id="products" class="row" data-hook>
     <% products.each do |product| %>
-      <% url = spree.product_url(product, taxon_id: @taxon.try(:id)) %>
+      <% url = spree.product_path(product, taxon_id: @taxon.try(:id)) %>
```

Comment:
You should probably update the local variable as well. It still reads `url`. Maybe `path` would be better.